tests/testthat/test-module_teal.R

```diff
@@ -83,8 +83,8 @@ testthat::describe("srv_teal lockfile", {
           ),
           expr = {
             iter <- 1
-            while (!file.exists(renv_filename) && iter <= 1000) {
-              Sys.sleep(0.5)
+            while (!file.exists(renv_filename) && iter <= 2000) {
+              Sys.sleep(1)
```

Comment:
So you extended the total wait time from 0.5*1000 seconds (8.3 minutes) to 1*2000 seconds (33 minutes) and it still does not work : ) ?

If it does not work for 8 minutes, it will not work at all. lockfile should take about 10-60 seconds in this case.

Reply:
Okay, let's investigate this more.